clamav/clamav.lua

```
@@ -1,318 +1,297 @@
-local sha512 = require("resty.sha512")
-local str = require("resty.string")
-local upload = require("resty.upload")
 local utils = require("bunkerweb.utils")
 
 local clamav = class("clamav", plugin)
 
 local ngx = ngx
-local NOTICE = ngx.NOTICE
 local ERR = ngx.ERR
-local socket = ngx.socket
+local NOTICE = ngx.NOTICE
```

Luacheck (1.2.0)

[warning] unused variable 'NOTICE'

(W211)